torch/csrc/jit/python/script_init.cpp

```diff
@@ -1684,6 +1685,9 @@ void initJitScriptBindings(PyObject* module) {
         }
         return _load_for_mobile(in, optional_device);
       });
+  m.def("_get_runtime_bytecode_version", []() {
```

Comment:
Python API for "lite interpreter runtime version" looks a little confusing to me. "runtime" is mostly used as a binary that can run on client. It's also the major purpose for this API. Adding a Python API means something from server side (because we don't assume Python be available on client). 

This API is telling the user which bytecode version the produced *model* would be,  not the actual version number deployed to the client. IMHO its good to expose to python the model API (provided a model file returns the version number inside that model), but not the runtime API, at least not with the name "runtime". Any thoughts @raziel ?

Reply:
I was thinking this api can be use by client right before the model is loaded in client. They can do a quick bytecode compatibility check before the model is loaded. Maybe rename it to `_get_library_bytecode_version`?

Reply:
My major concern is not on the naming. I don't think we should expose this API in python. If it's used in client, it's C++, not Python.

Reply:
Remove it from Python in latest commit.

Reply:
> My major concern is not on the naming. I don't think we should expose this API in python. If it's used in client, it's C++, not Python.

I agree with this with one caveat. If we ever have lite runtime deployed via python client. This might be the case in server but not usually the case in client. So I would still agree with @iseeyuan. If anything we might want to expose this via Java binding as well, if android apps want to convey the bytecode version that lite runtime can handle to model delivery.